### PR TITLE
feat: 🎸 host read/del/update capability

### PR DESCRIPTION
**Pull request by DhariniJeeva**

### CODE REVIEW

File: ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/hosts/index.hbs [hunk 1]
```diff
@@ -13,12 +13,14 @@
...
-              {{host.displayName}}
-            </LinkTo>
+            {{#if (can 'read model' host)}}
+              <LinkTo
+                @route='scopes.scope.host-catalogs.host-catalog.hosts.host'
+                @model={{host.id}}
+              >
+                {{host.displayName}}
+              </LinkTo>
+            {{/if}}
```
Comment (randallmorey) [inline ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/hosts/index.hbs:23]: Does this need an else clause?

**Review by randallmorey — CHANGES_REQUESTED** [review]

File: ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/hosts/index.hbs (hunk 1 above)
Comment (DhariniJeeva) [reply ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/hosts/index.hbs:23]: oops, it does! will update this and the other two PRs

File: ui/admin/tests/acceptance/host-catalogs/hosts/read-test.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    projectScope: null,
+    hostCatalogs: null,
+    hostCatalog: null,
+    hosts: null,
+    host: null,
+    unknownHost: null,
+    newHost: null,
+  };
+
+  hooks.beforeEach(function () {
```
Comment (calcaide) [inline ui/admin/tests/acceptance/host-catalogs/hosts/read-test.js:38]: @DhariniJeeva @randallmorey @gsusmi 
I am writing this as a question, not than as a statement: I see a recurrent pattern on all the capabilities tests. We are creating a test file per capability, which is a nice separation of concerns. But we are also repeating code through these files, all the instances, urls and setting those within the beforeEach statements are the same across capability.
I wonder if we could create a test helper file per capability where we define and export the repeating patterns, so then the test file just import those and we avoid repeating code.

For this use case, could be something like read-capability-test-helper.js? exports instances and urls initialization (basically generating the object equal to null) and generate all the resources.

Let me know your thoughts, perhaps this is not a good idea.

**Review by calcaide — APPROVED** [review]

File: ui/admin/tests/acceptance/host-catalogs/hosts/read-test.js (hunk 2 above)
Comment (randallmorey) [reply ui/admin/tests/acceptance/host-catalogs/hosts/read-test.js:38]: This is something I've thought about as well.  Essentially we need a standard way to model a test scenario and all the resources that go into it.  And from there, we might be able to move the standard test cases themselves into helpers.  This is housekeeping and hopefully we can find time in the future.

**Review by randallmorey — APPROVED** [review]

File: ui/admin/tests/acceptance/host-catalogs/hosts/read-test.js (hunk 2 above)
Comment (calcaide) [reply ui/admin/tests/acceptance/host-catalogs/hosts/read-test.js:38]: Sure! yeah I should mentioned before this message was OUT of topic of this PR and more a housekeeping thing.
Thanks!

Comment (DhariniJeeva) [reply ui/admin/tests/acceptance/host-catalogs/hosts/read-test.js:38]: Thanks for bringing this up @calcaide.

**Pull request merged**